src/main/java/cloud/eppo/ConfigurationRequestor.java

```diff
@@ -12,28 +14,44 @@ public class ConfigurationRequestor {
 
   private final EppoHttpClient client;
   private final ConfigurationStore configurationStore;
+  private final Set<String> loadedBanditModelVersions;
 
   public ConfigurationRequestor(ConfigurationStore configurationStore, EppoHttpClient client) {
     this.configurationStore = configurationStore;
     this.client = client;
+    this.loadedBanditModelVersions = new HashSet<>();
   }
 
+  // TODO: async loading for android
   public void load() {
     log.debug("Fetching configuration");
-    Response response = client.get("/api/flag-config/v1/config");
+    String flagConfigurationJsonString = requestBody("/api/flag-config/v1/config");
+    configurationStore.setFlagsFromJsonString(flagConfigurationJsonString);
```

Comment:
Do you need to worry about threading here and lock reads until the bandit models are set below?

Reply:
Good thinking! The underlying implementation is a thread-safe ConcurrentHashMap, which is what we are using in the Android SDK without (known) issues